fireapi/model/punkttyper.py

```diff
@@ -22,7 +22,10 @@ class Punkt(FikspunktregisterObjekt):
     geometriobjekter = relationship(
         "GeometriObjekt", order_by="GeometriObjekt.objectid", back_populates="punkt"
     )
-    # TODO: Observationer, PunktInfoer
+    punktinformationer = relationship(
+        "PunktInformation", order_by="PunktInformation.objectid", back_populates="punkt"
+    )
+    # TODO: Observationer (Note Observation has three references to Punkt. How to handle this?)
```

Comment:
Maybe use: "observationer_til" referencing Observation.Sigtepunkt and "observationer_fra" referencing Observation.opstillingspunkt